manimlib/mobject/mobject.py

```diff
@@ -82,6 +82,14 @@ def __init__(self, **kwargs):
     def __str__(self):
         return self.__class__.__name__
 
+    def __add__(self, other : 'Mobject') -> 'Mobject':
+        assert(isinstance(other, Mobject))
+        return self.get_group_class(self, other)
```

Comment:
```suggestion
        return self.get_group_class()(self, other)
```

Otherwise, this doesn't quite work, since Mobject.get_group_class returns the class